kotlin/internal/jvm/android.bzl

```diff
@@ -28,6 +28,7 @@ def _kt_android_artifact(
         kotlinc_opts = None,
         javac_opts = None,
         enable_data_binding = False,
+        testonly = False,
```

Comment:
Not a big deal, but I don't think adding this is needed - both the native `android.library` and the `_kt_jvm_library` fetch it from the `kwargs` anyway. The only change really needed here is the one on line 101.

Reply:
Tried the suggested fix to line 101, works as intended. Is there any way to speed up the process of applying these changes (like contributing to a subject PR)?